src/Game.cpp

```diff
@@ -39,17 +39,179 @@ static void sub_53A890(const char *args) {
     sub_612A80(loc_6C3690);
 }
 
+// mc2: 0x005EDA50
+void game_set_window_title(char* title) {
+    HWND hWnd = global_hWnd;
+    global_WindowText = title;
+    if (hWnd != NULL)
+        SetWindowTextA(global_hWnd, title);
+}
+
+// mc2: 0x00539FE0 and 0x00539D90
+void check_sku_version() {
+    global_SKUVersion = 0; //SKU VERSION SLUS-20209
+
+    char* skuVersion = NULL;
+
+    switch (global_SKUVersion) {
```

Comment:
From my research, these appear to be PlayStation 2 version codes. This seems to tell me that there's a lot of code shared between versions (they may even share the same codebase using the preprocessor). 

As to this method as a whole, while setting `global_SKUVersion` is a good idea (since other code seems to be doing... something... with it). However, we may want to either comment out the code that prints the SKU, or just comment out the code getting the string for the SKU and replace it with an OpenMC2-specific string. Not that it's likely to cause an issue, but I just think it would look cleaner to do that.